.github/workflows/pr.yml

```
@@ -344,11 +344,32 @@ jobs:
           echo "Updated Comment Body: $UPDATED_BODY"
 
   publish-to-prod-on-pr-merge:
-    if: ${{ github.event_name == 'pull_request' && github.event.action == 'closed' && github.event.pull_request.merged == true }}
+    # TEMPORARILY COMMENTED OUT TO TEST API REQUEST
+    # if: ${{ github.event_name == 'pull_request' && github.event.action == 'closed' && github.event.pull_request.merged == true }}
```

I'll need to uncomment this once this PR is reviewed/accepted. But I have it uncommented to demonstrate what the job does
```suggestion
    if: ${{ github.event_name == 'pull_request' && github.event.action == 'closed' && github.event.pull_request.merged == true }}
```